views/inventory.hbs

```diff
@@ -128,6 +129,18 @@
       <tbody>
         {{#each this}}
           {{#if name}}
+            {{#if not_catalog}}
```

Comment:
```suggestion
            {{#unless active}}
```
This would proceed through the conditional if `active is false`. This might eliminate the need to add a new property `not_catalog`?